source/_docs/eu-region.md

```diff
@@ -30,40 +30,39 @@ Once enabled, this Organization-wide feature allows <a href="/docs/change-manage
 
 ### EU Data Residency
 
-Now you can run WordPress or Drupal sites on Pantheon and meet legal, regulatory, or data sovereignty requirements from the European Union.
-
-Pantheon sites running in Europe have all site resources in the EU. This includes application and database containers, distributed filesystem and request router, Redis cache servers, and Apache Solr index servers.
+Pantheon sites running in Europe have all site resources in the EU. This includes application and database containers, Redis cache servers, Apache Solr index servers, and a distributed filesystem and request router.
 
 Automated and manual backups of all site components (code, database, and files) are stored in the EU and created by job workers also running in the EU. Additionally, any database or file clones between site environments are run by EU job workers.
 
+With this set of EU-specific resources, now you can run WordPress or Drupal sites on Pantheon and meet the European Union's legal, regulatory, or data sovereignty requirements.
+
 ## Create a New Site
 
-1. Install and authenticate [Terminus 2.0](/docs/terminus/). The commands used for International Regions require Terminus 2.0. If you're already running Terminus, be sure and update to the [latest version](/docs/terminus/updates/).
-2. Use Terminus to create a new site associated with your organization and include the `--region=eu` option.
+1. Install and authenticate [Terminus](/docs/terminus/). The commands used for International Regions require Terminus 2.0 or newer. If you're already running Terminus, be sure to update to the [latest version](/docs/terminus/updates/).
+1. Use Terminus to create a new site associated with your organization and include the `--region=eu` option.
 
  For example (replace `my-eu-site-name`, `My EU Site Name`, `WordPress` and `My Organization Name` accordingly):
 
- <div class="copy-snippet">
-   <button class="btn btn-default btn-clippy" data-clipboard-target="#terminus-new-site">Copy</button>
-   <figure><pre id="terminus-new-site"><code class="command bash" data-lang="bash">
-  terminus site:create my-eu-site-name "My EU Site Name" "WordPress" --org "My Organization Name" --region eu</code></pre></figure>
- </div>
+ ```bash
+ terminus site:create my-eu-site-name "My EU Site Name" "WordPress" --org "My Organization Name" --region eu
+ ```
 
   ![terminus site:create my-eu-site "My EU Site" "WordPress" --org "Rachel Pantheor" --region eu](/source/docs/assets/images/create-site-eu.png)
 
+  See `terminus site:create --help` for more information on the options and values used in this command.
```

Comment:
@EdwardAngert this is the only edit I think needs your +1.

Reply:
👍